### ISSUE TITLE
cant run butler-sos in docker, clean setup

### ISSUE THREAD

**Issue by bintocher** [ptarmiganlabs/butler-sos#594]
### What version of Butler SOS are you using?

docker image ptarmiganlabs/butler-sos:latest

### What version of Node.js are you using? Not applicable if you use the standalone version of Butler SOS.

docker image ptarmiganlabs/butler-sos:latest

### What command did you use to start Butler SOS?

docker-compose up

### What operating system are you using?

windows 11 pro ver 22621.2134

### What CPU architecture are you using?

x86_x64

### What Qlik Sense versions are you using?

2023-february sr2

### Describe the Bug

docker-compose.yml I take from here https://butler-sos.ptarmiganlabs.com/docs/getting_started/install/docker/

I make clean setup (no images, no containers in docker), and give error, in first run, so, butler not start

![image](https://github.com/ptarmiganlabs/butler-sos/assets/8188055/7434b4ad-cab6-4d93-9ee4-5e9cbc25698d)

```

version: "3.3"
services:
    butler-sos:
        image: ptarmiganlabs/butler-sos:latest
        container_name: butler-sos
        restart: always
        volumes:
            # Make config file and log files accessible outside of container
            - "./config:/nodeapp/config"
            - "./log:/nodeapp/log"
        environment:
            - "NODE_ENV=production" # Means that Butler SOS will read config data from production.yaml
        logging:
            driver: "json-file"
            options:
                max-file: "5"
                max-size: "5m"
        networks:
            - senseops

networks:
    senseops:
        driver: bridge

```

error i get:
```
docker-compose up  
[+] Running 1/0
 ✔ Container butler-sos  Created                                                                                                                                                                                                                                                                       0.0s 
Attaching to butler-sos
butler-sos  | node:internal/modules/cjs/loader:1093
butler-sos  |   throw err;
butler-sos  |   ^
butler-sos  | 
butler-sos  | Error: Cannot find module '../package.json'
butler-sos  | Require stack:
butler-sos  | - /nodeapp/globals.js
butler-sos  | - /nodeapp/butler-sos.js
butler-sos  |     at Module._resolveFilename (node:internal/modules/cjs/loader:1090:15)
butler-sos  |     at Module._load (node:internal/modules/cjs/loader:934:27)
butler-sos  |     at Module.require (node:internal/modules/cjs/loader:1157:19)
butler-sos  |     at require (node:internal/modules/helpers:119:18)
butler-sos  |     at Object.<anonymous> (/nodeapp/globals.js:27:20)
butler-sos  |     at Module._compile (node:internal/modules/cjs/loader:1275:14)
butler-sos  |     at Module._extensions..js (node:internal/modules/cjs/loader:1329:10)
butler-sos  |     at Module.load (node:internal/modules/cjs/loader:1133:32)
butler-sos  |     at Module._load (node:internal/modules/cjs/loader:972:12)
butler-sos  |     at Module.require (node:internal/modules/cjs/loader:1157:19) {
butler-sos  |   code: 'MODULE_NOT_FOUND',
butler-sos  |   requireStack: [ '/nodeapp/globals.js', '/nodeapp/butler-sos.js' ]
butler-sos  | }
butler-sos  |
butler-sos  | Node.js v19.9.0
butler-sos  |
butler-sos exited with code 0
```
### Expected Behavior

_No response_

### To Reproduce

_No response_

**Comment by mountaindude** [comment]
Please give [version 9.6.1](https://github.com/ptarmiganlabs/butler-sos/releases) a try. 

Available from Docker Hub as both `latest`and `9.6.1`.

**Comment by bintocher** [comment]
@mountaindude tnx, now work!
